nemo/collections/tts/models/fastpitch.py

```diff
@@ -251,7 +251,7 @@ def training_step(self, batch, batch_idx):
                 self.global_step,
                 dataformats="HWC",
             )
-            spec_predict = mels_pred[0].data.cpu().numpy().T
+            spec_predict = mels_pred[0].data.cpu().numpy()
```

Comment:
Why undo the transpose?

Reply:
Spectrograms appeared transposed in tensorboard.

Reply:
Yea, let's go with your fix. Still not sure where that came from.